.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "eslint.options": {
+        "resolvePluginsRelativeTo": "./packages/pwa-kit-dev",
+    },
+    "eslint.nodePath": "./packages/pwa-kit-dev/node_modules"
```

Comment:
To resolve eslint and its plugins, in the context of monorepo.

Reply:
If you simply want to have a workspace settings.json file for your own use (when working with the monorepo), here's a standalone file that has worked for me:

```json
{
  "eslint.options": {
    "resolvePluginsRelativeTo": "./packages/pwa-kit-dev",
    "configFile": "./packages/pwa-kit-dev/src/configs/eslint/eslint-config.js"
  },
  "eslint.nodePath": "./packages/pwa-kit-dev/node_modules"
}
```